lib/errors.ts

```diff
@@ -27,6 +27,40 @@ export class BalenaError extends TypedError {
 BalenaError.prototype.code = 'BalenaError';
 BalenaError.prototype.exitCode = 1;
 
+/**
+ * @summary Balena too many requests
+ * @class
+ * @public
+ *
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.BalenaTooManyRequests()
+ */
+export class BalenaTooManyRequests extends BalenaError {
+	constructor() {
```

Comment:
How about also adding
```suggestion
	constructor(public retryAfter?: string) {
```
?
Maybe we can do that later, since I'm not sure how we could add it in the message.

Reply:
@thgreasi I thought about the same thing, but I was thinking to add it later when this would be more clear. Just to be sure, do you mean `retryAfter` as a date string ?

Reply:
No strong opinion on this, but yea that what I meant, since (I think) that's what the API returns.
If you prefer leaving this for later that's fine with me.